Objects/codeobject.c

```diff
@@ -452,10 +452,15 @@ static PyObject *
 code_sizeof(PyCodeObject *co, void *unused)
 {
     Py_ssize_t res;
-
     res = _PyObject_SIZE(Py_TYPE(co));
```

Comment:
If remove an empty line between declaration and assignment it is worth to merge them in one declaration with initialization.